packages/core/platform/Makefile

```
@@ -1,23 +1,20 @@
-
 show:
-	helm template -n $(NAMESPACE) $(NAME) . --dry-run=server $(API_VERSIONS_FLAGS)
+	cozypkg show -n $(NAMESPACE) $(NAME) --plain
 
 apply:
-	helm template -n $(NAMESPACE) $(NAME) . --dry-run=server $(API_VERSIONS_FLAGS) \
-		| kubectl apply -f-
+	cozypkg show -n $(NAMESPACE) $(NAME) --plain | kubectl apply -f-
 	kubectl delete helmreleases.helm.toolkit.fluxcd.io -l cozystack.io/marked-for-deletion=true -A
 
 reconcile: apply
```

checkmake (0.2.2)

[warning] Target "reconcile" should be declared PHONY.

(phonydeclared)